### ISSUE TITLE
Response header of JdkHttpConnector plugin is saved twice.

### ISSUE THREAD

**Issue by emeroad** [pinpoint-apm/pinpoint#8159]
![image](https://user-images.githubusercontent.com/7564547/130779470-489b5845-585e-41d4-9dae-8a8118129720.png)

Add `-Dprofiler.http.record.response.headers=Connection` option when starting JdkHttpPluginTestStarter.

HttpURLConnectionIT also fails the test.
```java
public class HttpURLConnectionIT {
    @Test
    public void test() throws Exception {
        URL url = new URL(webServer.getCallHttpUrl());
        HttpURLConnection connection = (HttpURLConnection)url.openConnection();
        connection.getHeaderFields();
        
        PluginTestVerifier verifier = PluginTestVerifierHolder.getInstance();
        verifier.printMethod();
        
        Class<?> targetClass = Class.forName("sun.net.www.protocol.http.HttpURLConnection");
        Method getInputStream = targetClass.getMethod("getInputStream");
        
        String destinationId = webServer.getHostAndPort();
        String httpUrl = webServer.getCallHttpUrl();
        verifier.verifyTraceCount(2);
        verifier.verifyTrace(event("JDK_HTTPURLCONNECTOR", getInputStream, null, null, destinationId,
                annotation("http.url", httpUrl),
                annotation("http.resp.header", anyAnnotationValue()),
                annotation("http.resp.header", anyAnnotationValue())  // <----- Duplicated res header 
                )
        );
    }
```

Related issue : https://github.com/pinpoint-apm/pinpoint/issues/7962

**Comment by emeroad** [comment]
@yjqg6666 
Could you please take a look at this issue.

**Comment by emeroad** [comment]
@yjqg6666 
test : HttpURLConnectionIT.testConnectTwice
The above test is still unsuccessful. could you please take a look

**Comment by yjqg6666** [comment]
@emeroad Checking.

**Comment by yjqg6666** [comment]
@emeroad Need your help.

> 09-28 16:09:56.056 [:child:8 Thread] DEBUG c.n.p.p.c.s.BufferedStorage              -- Flush SpanChunk{traceRoot=DefaultTraceRoot{traceId=DefaultTraceId{agentId='build.test.0', agentStartTime=1632816593420, transactionSequence=1, parentSpanId=-1, spanId=-3337857124470666767, flags=0}, agentId='build.test.0', traceStartTime=1632816596792}, spanEventList=[SpanEvent{stackId=-1, timeRecording=true, startTime=1632816596887, elapsedTime=10, asyncIdObject=null, sequence=1, serviceType=9055, endPoint='null', annotations=[StringAnnotation{40=http://localhost:51578}, DataTypeAnnotation{55=StringStringValue{stringValue1='Connection', stringValue2='keep-alive'}}], depth=-1, nextSpanId=-1, destinationId='localhost:51578', apiId=-2, exceptionInfo=null} ]}

The response header annotation is recorded, but with different type "DataTypeAnnotation" other than "StringAnnotation".
"DataTypeAnnotation{55" is the http.resp.header annotation.
But the verifier can not get the annotation, so failed. Am I check it in a incorrect way?

>      verifier.verifyTrace(event("JDK_HTTPURLCONNECTOR", connect, null, null, destinationId,
>                annotation("http.url", httpUrl),
>                annotation("http.resp.header", anyAnnotationValue())));

>  AnnotationKey HTTP_RESPONSE_HEADER = AnnotationKeyFactory.of(55, "http.resp.header", VIEW_IN_RECORD_SET);

**Comment by yjqg6666** [comment]
@emeroad Fixed by #8278. The plugin-it should be updated to check getInputStream method if the HttpURLConnection.getInputStream() get called. I just known that after looking into PluginTestVerifier.verifyTrace(). My bad.

**Comment by emeroad** [comment]
@yjqg6666 Thnaks!
